src/govuk/components/select/_index.scss

```diff
@@ -15,6 +15,11 @@
     padding: govuk-spacing(1); // was 5px 4px 4px - size of it should be adjusted to match other form elements
     border: $govuk-border-width-form-element solid $govuk-input-border-colour;
 
+    // Sets the background and foreground colours
```

Comment:
Minor point, but given this is a relatively 'expected' thing to do, and we have the Git history, I do wonder how useful the comment is.

If we do want to include a comment, it might be worth including a summary of the issue we're trying to fix, rather than requiring the reader to go to the GitHub issue for the context?

Perhaps something like:

```scss
    // Default user agent colours for selects can have low contrast, and may
    // look disabled (#2435)
```

Reply:
I think I just wanted a comment in there because otherwise it felt easy for someone to remove (for example) the text color without realising why it was being set. But that's a good point, I've added your suggestion 👍